Yoast/wordpress-seo
# Make the pluginicon in the editor reflect the SEO and Readability score.

## Context


* To increase discoverability of the Yoast sidebar / editor integration, it would be nice if the Yoast icon would reflect the SEO and Readability scores. This PR adds that:

<img width="428" alt="Screenshot 2020-06-15 at 17 35 43" src="https://user-images.githubusercontent.com/1488816/84677198-b1871d80-af2e-11ea-9f08-681d404da71f.png">


## Summary


This PR can be summarized in the following changelog entry:

* Makes the pluginicon in the editor reflect the SEO and Readability score.

## Relevant technical choices:

* Chose to inject the colors for the bullets using styled components as it is much easier to update the styles based on the Redux state than it is to update the classes on those SVG circles. 

## Test instructions

This PR can be tested by following these steps:

* Try to get a few variations on a post in terms of SEO and readability scores.
* Notice the icon changes in accordance with the scores you get.
* Disable the readability analysis / seo analysis in different combinations.
* Notice the icon takes that into account.

## UI changes

## Documentation

## Quality assurance
